packages/visx-text/src/Text.tsx

```diff
@@ -111,6 +117,12 @@ class Text extends React.Component<TextProps, TextState> {
     this.updateWordsByLines(this.props, needCalculate);
   }
 
+  updateTextPathId(props: TextProps) {
+    if(props.textPath) {
+      this.setState({ textPathId: uniqueId('text-path-') })
+    }
```

Comment:
I see this is called from `componentDidMount()`. Ideally we could move this to the `constructor()` because it doesn't rely on the DOM.

Reply:
Sorry, it took me so long to fix this. I've moved the whole state initialization into the constructor with the id generation in there as well.